specification/datafactory/resource-manager/Microsoft.DataFactory/preview/2017-09-01-preview/entityTypes/Trigger.json

```diff
@@ -365,6 +372,24 @@
           "maximum": 86400
         }
       }
+    },
+    "TumblingWindowDependency": {
+      "description": "Tumbling Window dependency information.",
+      "type": "object",
+      "properties": {
+        "type": {
+          "description": "Reference type \"TriggerReference\" for Tumbling Window.",

```

Comment:
Suggested description: "Tumbling window trigger reference type." This is consistent with our other descriptions of similar kind.

Reply:
Resolved